ibm_mq/setup.py

```diff
@@ -18,7 +18,7 @@
     long_description = f.read()
 
 
-CHECKS_BASE_REQ = 'datadog-checks-base>=4.2.0'
+CHECKS_BASE_REQ = 'datadog-checks-base>=11.0.0'
```

Comment:
Any reason to upgrade to `11.0.0`? Should we target the lowest compatible `datadog-checks-base` version?

Reply:
Adding specs requires latest agent because it includes service tag